worlds/wargroove2/Rules.py

```diff
@@ -0,0 +1,32 @@
+from typing import TYPE_CHECKING
+from worlds.AutoWorld import LogicMixin
+from .Levels import first_level
+if TYPE_CHECKING:
+    from . import Wargroove2World
+
+
+class Wargroove2Logic(LogicMixin):
+    pass
```

Comment:
LogicMixin is crazy so maybe this is needed, but if it isn't, you should probably remove it.